tests/rptest/tests/compatibility/kafka_streams_test.py

```diff
@@ -550,3 +551,197 @@ def test_kafka_streams_map_function(self):
         consumer.stop()
         self._producer.stop()
         example.stop()
+
+
+class KafkaStreamsInteractiveQueries(RedpandaTest):
+    """
+    Test KafkaStreams InteractiveQueries app which locates and
+    queries kafka streams state stores. The state stores used in this app are
+    a KeyValue store and a Windowed Store.
+
+    Outside of KStreams, the app does simple prod-cons so the used RP features and systems are:
+    - topic creation
+    - produce/consume on topic
+    - src/v/kafka
+    - src/v/raft
+    - src/v/storage
+    """
+
+    topics = (TopicSpec(name="TextLinesTopic"), )
+
+    def __init__(self, test_context):
+        super(KafkaStreamsInteractiveQueries,
+              self).__init__(test_context=test_context)
+        self._ctx = test_context
+        self._ports = [7070, 7071]
+        self._store_names = ["windowed-word-count", "word-count"]
+
+    def list_all_running_instances(self, res):
```

Comment:
`list_xxx` implies that the method list something and the return type is a `list` but this method validates and returns boolean. can you rename it into something like `check_xxx` or `validate_xxx` to give a strong signal of what it does.